yt_dlp/extractor/neteasemusic.py

```diff
@@ -203,91 +165,148 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'upload_date': '19911130',
             'timestamp': 691516800,
             'description': 'md5:1ba2f911a2b0aa398479f595224f2141',
+            'subtitles': {'lyric': [{'ext': 'lrc'}]},
+            'duration': 268,
+            'alt_title': '伴唱:现代人乐队 合唱:总政歌舞团',
+            'thumbnail': r're:^http.*\.jpg',
+        },
+    }, {
+        'url': 'http://music.163.com/#/song?id=32102397',
+        'md5': '3e909614ce09b1ccef4a3eb205441190',
+        'info_dict': {
+            'id': '32102397',
+            'ext': 'mp3',
+            'title': 'Bad Blood',
+            'creator': 'Taylor Swift / Kendrick Lamar',
+            'upload_date': '20150516',
+            'timestamp': 1431792000,
+            'description': 'md5:21535156efb73d6d1c355f95616e285a',
+            'subtitles': {'lyric': [{'ext': 'lrc'}]},
+            'duration': 199,
+            'thumbnail': r're:^http.*\.jpg',
+        },
+        'skip': 'Blocked outside Mainland China',
+    }, {
+        'note': 'Has translated name.',
+        'url': 'http://music.163.com/#/song?id=22735043',
+        'info_dict': {
+            'id': '22735043',
+            'ext': 'mp3',
+            'title': '소원을 말해봐 (Genie)',
+            'creator': '少女时代',
+            'upload_date': '20100127',
+            'timestamp': 1264608000,
+            'description': 'md5:79d99cc560e4ca97e0c4d86800ee4184',
+            'subtitles': {'lyric': [{'ext': 'lrc'}]},
+            'duration': 229,
+            'alt_title': '说出愿望吧(Genie)',
+            'thumbnail': r're:^http.*\.jpg',
         },
+        'skip': 'Blocked outside Mainland China',
     }]
 
     def _process_lyrics(self, lyrics_info):
-        original = lyrics_info.get('lrc', {}).get('lyric')
-        translated = lyrics_info.get('tlyric', {}).get('lyric')
+        original = traverse_obj(lyrics_info, ('lrc', 'lyric', {str}))
+        translated = traverse_obj(lyrics_info, ('tlyric', 'lyric', {str}))
+
+        if not original or original == '[99:00.00]纯音乐，请欣赏\n':
+            return None
 
         if not translated:
-            return original
+            return {
+                'lyrics': [{'data': original, 'ext': 'lrc'}],
+                'lyrics_original': [{'data': original, 'ext': 'lrc'}],
```

Comment:
I'm not sure I like this... This will download the same subtitles twice if we download all subs. Opinions?

Reply:
Should we just remove the second line then?
```suggestion
                'lyrics': [{'data': original, 'ext': 'lrc'}],
```

Reply:
That would work, but maybe we should flip it then? lyrics should aleays be original and merged would become `lyrics-merged` below? That way there is no mix between merged and non merged? Or would you want that so you can download the merged in case they exist? That behavior would be opinionated but would work since merging only adds info. Ill leave to your decision

Reply:
That's a good point.
The way that previous code prioritize merged lyrics looked intuitive to me, but we are not limited to a single option when writing to files. `lyrics-merged` is indeed a better option when it comes to subititle files.
We can keep the description to merged first, original second, while keeping the subtitle files in a clearer way.

Reply:
https://github.com/yt-dlp/yt-dlp/pull/8181/commits/dae7a60b805ac2ae50aeeb1d5e7f0110f8babbb5